Jetcaster/gradle/libs.versions.toml

```diff
@@ -58,6 +58,10 @@ secrets = "2.0.1"
 # @keep
 targetSdk = "33"
 version-catalog-update = "0.8.4"
+glanceAppwidget = "1.1.0-beta02"
+glanceMaterial3 = "1.1.0-beta02"
+glance = "1.1.0-beta02"
+composeBom = "2024.04.01"
```

Comment:
Changes to the toml file need to happen in the top level one as they are duplicated across samples.

Reply:
I've checked it. :)